pages/apply-dev.html

```diff
@@ -33,7 +33,8 @@ <h3 class="info-header">What You&rsquo;ll be doing</h3>
 
   <section class="info">
     <h3 class="info-header">Applications</h3>
-    <p class="info-secondary">Applications for our 2021 Full-Stack Developer Apprenticeship will open in fall of 2020. Be sure to <a href="https://twitter.com/hearsparkbox">follow us on Twitter</a> and check <a href="https://seesparkbox.com/foundry/">The Foundry</a> for updates.</a
+     <p class="info-secondary">If you feel like you’d be a great fit for our remote full-stack apprenticeship, we encourage you to apply.</p>	    
+  <a href="https://sparkbox.workable.com/jobs/1408239" target="_blank" class="button">Begin Application</a>
```

Comment:
Is it okay if I scooch this fella to aline it with sibling tags?
```suggestion
    <a href="https://sparkbox.workable.com/jobs/1408239" target="_blank" class="button">Begin Application</a>
```